src/actions.ts

```diff
@@ -0,0 +1,10 @@
+// eslint-disable-next-line import/no-extraneous-dependencies
+import { createAction } from 'redux-act';
+
+export const setRedModEnable = createAction(`SET_REDMOD_ENABLE`, (enabled) => enabled);
+export const setAutoRun = createAction(`SET_AUTO_RED_DEPLOY`, (enabled) => enabled);
+export const setArchiveAutoConvert = createAction(`SET_ARCHIVE_AUTO_CONVERT`, (enabled) => enabled);
+export const setNeedToRun = createAction(
```

Comment:
Minor: these should correspond exactly to feature naming (or vice versa) plus prefixes/suffixes as needed to distinguish type/context.